src/Tests/NugetPackagesTests.cs

```diff
@@ -84,7 +84,7 @@ public async Task FindDependencies()
             using (var context = new SourceCacheContext())
             {
                 await mgr.FindDependencies(pkgId, context);
-                Assert.AreEqual(163, mgr.AvailablePackages.Count());
```

Comment:
Are these reasonable tests?

Reply:
I'm not fully sure I get the exact point of the question.

During debugging I got an impression that the tests count the number of project dependencies and compare to the expected number.